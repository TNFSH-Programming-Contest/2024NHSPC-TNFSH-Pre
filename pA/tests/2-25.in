375995019667170444 616730211876934857 978848660612616052
36498697656986507 999020612854306903 936054581645427943
