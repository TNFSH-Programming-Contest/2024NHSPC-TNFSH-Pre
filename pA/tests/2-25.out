408516866646469164
